manifests/v/VSRevoGroup/RevoUninstallerPro/5.0.1.0/VSRevoGroup.RevoUninstallerPro.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: VSRevoGroup.RevoUninstallerPro
+PackageVersion: 5.0.1.0

```

```suggestion
PackageVersion: 5.0.1
```
Should match control panel